src/frontend/components/admin/Glossary.jsx

```diff
@@ -315,7 +316,7 @@ export default function EnhancedTable(props) {
                           <TableCell>{row.term}</TableCell>
                           <TableCell>
                             <Truncate lines={1} width={300}>
-                              {row.definition}
+                              {parse('<div>' + row.definition + '</div>')}
```

Comment:
In my local build, the parser isn't working here :( It works when I comment out the `<Truncate>` component. Ugh.